search-service/src/test/kotlin/com/egm/stellio/search/entity/web/EntityHandlerTests.kt

```diff
@@ -480,6 +480,47 @@ class EntityHandlerTests {
             )
     }
 
+    @Test
+    fun `get entity by id should correctly return the representation asked in the format parameter`() {
```

Comment:
better add one or more tests on the `parseRepresentations` function in `NgsiLdDataRepresentation`